Tools/autotest/ardusub.py

```diff
@@ -474,6 +475,46 @@ def MAV_CMD_DO_CHANGE_SPEED(self):
                 self.wait_groundspeed(speed-0.2, speed+0.2, minimum_duration=2, timeout=60)
         self.disarm_vehicle()
 
+    def GPSForYaw(self):
+        '''Moving baseline GPS yaw'''
+        # store previous heading
+        previous_heading = self.assert_receive_message("ATTITUDE").yaw
+
+        # load parameters with gps for yaw and 10 degrees offset
+        self.context_push()
+        self.load_default_params_file("sub-gps-for-yaw-nmea.parm")
+        self.reboot_sitl()
+        ex = None
+        try:
```

Comment:
You don't need this try block; if the test fails there's an implicit reboot now-adays

Reply:
nice, thanks!